vitest.setup.ts

```diff
@@ -55,18 +60,112 @@ if (version.startsWith("17")) {
   });
 }
 
-beforeEach(async ({ task }) => {
-  const filename = task.file?.name;
-  if (!filename) return;
-  const match = filename.match(/examples\/(.*)\/test.ts$/);
-  if (!match) return;
-  const [, example] = match;
-  const { default: comp } = await import(`./examples/${example}/index.tsx`);
-  const element = createElement(Suspense, {
+const ALLOWED_TEST_LOADERS = ["react", "solid"] as const;
+type AllowedTestLoader = (typeof ALLOWED_TEST_LOADERS)[number];
+declare global {
+  var loader: AllowedTestLoader;
+}
+const TEST_LOADER = (process.env.ARIAKIT_TEST_LOADER ??
+  "react") as AllowedTestLoader;
+globalThis.loader = TEST_LOADER;
```

Comment:
I noticed that the global `loader` variable is currently only used in the blur test. Are there any other use cases you're considering?